appcelerator-archive/amp
# Voting app example

This is the voting app example from docker (https://github.com/docker/example-voting-app) adapted to AMP.

## To test
```
amp cluster create
amp --server=localhost stack deploy -c examples/stacks/voting-app/voting.stack.yml voting
```

Then you should be able to reach:
 - the voting app:  https://vote.local.atomiq.io
 - the result app: https://result.local.atomiq.io
